modules/auth/sso/basic.go

```diff
@@ -52,11 +52,14 @@ func (b *Basic) IsEnabled() bool {
 func (b *Basic) VerifyAuthData(ctx *macaron.Context, sess session.Store) *models.User {
 	baHead := ctx.Req.Header.Get("Authorization")
 	if len(baHead) == 0 {
-		return nil
+		baHead = ctx.Req.Header.Get("authorization")
```

Comment:
This is not needed. Header.Get is case insensitivie

Reply:
It does already work with lowercase header name:
![image](https://user-images.githubusercontent.com/165205/102862953-8d0a8180-443a-11eb-80f2-383951e71152.png)